### DISCUSSION TITLE
encountered wrong indentation many time while defining a function

### DISCUSSION THREAD

**Discussion by SambhavKamboj** [community/community#68527] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I recently started using GitHub copilot
While writing a python program for my school project I encountered the indentation error many time
though its easy to solve but its pretty annoying to change indentation again again
So please checkout the issue
![Screenshot 2023-09-28 194359](https://github.com/community/community/assets/52831046/3d1eb78b-dfe7-4ec7-943e-226abf808f9a)

**Comment by Primus66** [comment]
It looks like you're experiencing an issue with incorrect indentation in your Python code while using GitHub Copilot. Based on the code snippet you provided, the indentation appears to be inconsistent, which can indeed cause errors in Python.

To fix the indentation in the code snippet you shared, here's the corrected version:

```python
if stream == "PCM":
    p = int(input("Enter marks in Physics out of 100: "))
    c = int(input("Enter marks in Chemistry out of 100: "))
    m = int(input("Enter marks in Maths out of 100: "))
    e = int(input("Enter marks in English out of 100: "))
    comp = int(input("Enter marks in Computer out of 100: "))
    total = p + c + m + e + comp
```

In Python, proper indentation is crucial for the code to execute correctly. Each block of code under an `if` statement or loop should be indented with consistent spacing (usually four spaces) to define its scope.

If GitHub Copilot is automatically generating incorrect indentation, you may want to check your code editor settings or extensions to ensure they are not conflicting with Copilot's suggestions. Additionally, ensure that your Python environment is correctly configured to use spaces for indentation, as Python is sensitive to indentation style.

**Reply by SambhavKamboj** [reply]
Thank You!
Though I know how to fix indentation error😊

**Reply by movahhedi** [reply]
The answer is AI-generated.
